dev/fattest.simplicity/src/componenttest/topology/utils/tck/TCKResultsWriter.java

```diff
@@ -74,7 +74,7 @@ public static void preparePublicationFile(TCKResultsInfo resultInfo) {
             specURL = "https://jakarta.ee/specifications/" + specName + "/" + specVersion;
             tckURL = "https://download.eclipse.org/ee4j/" + specName + "/jakartaee10/promoted/eftl/" + specName + "-tck-" + specVersion + ".zip"; //just a placeholder, needs to be manually updated
         }
-        String filename = openLibertyVersion + "-" + fullSpecName.replace(" ", "-") + "-Java" + javaMajorVersion + "-TCKResults.adoc";
+        String filename = openLibertyVersion + "-" + fullSpecName.replace(" ", "-") + "-Java" + javaMajorVersion + "-TCKResults" + repeat + ".adoc";
```

Comment:
Does the repeat string already include a separator char? Should a `-` be added after `-TCKResults`. What happens if the repeat string is null?

Reply:
Yes it includes `_` . But in the new commit I have made it so all the `_` are replaced with `-` to keep it consistent with the `filenames`.